.evergreen/config_generator/components/funcs/find_cmake_latest.py

```diff
@@ -0,0 +1,25 @@
+from shrub.v3.evg_command import EvgCommandType
+
+from config_generator.etc.function import Function
+from config_generator.etc.utils import bash_exec
+
+
+class FindCMakeLatest(Function):
```

Comment:
```suggestion
class FindCMakeLatest(Function):
    """
    Pre-fetch the latest CMake with retry.
    Call in tasks requiring CMake for a more reliable download.
    """
```

I suggest renaming to clarify the intent is to download CMake reliably. I expect callers of this function need to rerun `find-cmake-latest.sh` to locate the downloaded CMake from cache.

Reply:
Added documentation clarifying intent. I would prefer to keep it named as-is to match the script/function it is meant to invoke.